src/main/resources/com/cloudbees/jenkins/support/config/SupportAutomatedBundleConfiguration/help-period.html

```diff
@@ -0,0 +1,4 @@
+<div>
```

Comment:
I'm wondering if there should be some kind of message here clarifying how this works with the system property.

Reply:
Good point! Will definitely add that.

Reply:
done